test/tutorial/scripts/api/get_bundles_checkout_api.py

```diff
@@ -0,0 +1,11 @@
+from hca import HCAConfig
+from hca.dss import DSSClient
+import sys
+
+hca_config = HCAConfig()
+hca_config["DSSClient"].swagger_url = f"https://dss.data.humancellatlas.org/v1/swagger.json"
+dss = DSSClient(config=hca_config)
+
+bundle_checkout_status = dss.get_bundles_checkout(
+        replica="aws", checkout_job_id='4de1c603-fa8b-4c07-af37-06159e6951e0')
+print( bundle_checkout_status)
```

Comment:
Extra space and EOF.  Please run: `flake8 test/tutorial/scripts/api/get_bundles_checkout_api.py` and you will see the error.  Also, remember to run `flake8` for the entire tutorial folder (`flake8 test/tutorial/scripts`).